src/sisyphus/rabbit.clj

```diff
@@ -107,11 +122,20 @@
   (let [message (json/parse-string raw true)]
     (log/info! "rabbit message received:" message)))
 
+(def parse-options
+  [["-q" "--queue QUEUE" "queue to subscribe to"
+    :default "sisyphus-queue"]
+   ["-r" "--routing-key ROUTING_KEY" "routing key for messages"
+    :default "sisyphus-task"]
+   ["-e" "--exchange EXCHANGE" "message exchange"
+    :default "sisyphus-exchange"]])
```

Comment:
Yuk. Why doesn't this just call `logs/gce-metadata` to get each metadata field rather than getting it from the command line and making something else get these metadata fields just to feed them into on the command line?

Reply:
If the CLI args are for debugging outside GCE, we could adjust `default-config` or read defaults from a debug config file.

Reply:
Hey @1fish2! I am still planning on using the GCE metadata to set these values for Sisyphus. The main in this namespace (`sisyphus.rabbit`) is for testing purposes, I was using it to check out queue behavior with routing keys and exchanges etc. The main for sisyphus is in `sisyphus.core`, which will read from metadata once Gaia sets it on launch. I am planning to do that in another PR.

Reply:
I got it now. That's good.

Just please add a comment to `parse-options` and this other `-main` to say that these are for testing purposes.

Reply:
Done, thanks!

Reply:
Thanks! The testing feature is a great thing.